ckan/i18n/fr/LC_MESSAGES/ckan.po

```diff
@@ -1720,10 +1720,10 @@ msgstr "La longueur du mot-clé \"%s\" est supérieure à la longueur maximale %
 
```

Comment:
Please revert changes on this file. po files are managed on Transifex, and will get updated by translators once the base English string is updated